package.json

```diff
@@ -22,7 +22,8 @@
     "clean": "rm -rf lib && mkdir lib",
     "lib": "npm run clean && babel src -d lib",
     "docs": "node ./scripts/docs-server",
```

Comment:
Nit-picky, can you change the base "docs" command to `npm run docs-server` so that we don't have duplicate command code in there?